wpimath/src/test/java/edu/wpi/first/math/geometry/serde/Pose2dSerdeTest.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.geometry.serde;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.proto.Geometry2D.ProtobufPose2d;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import org.junit.jupiter.api.Test;
+
+class Pose2dSerdeTest {
+  private static final Pose2d DATA =
+      new Pose2d(new Translation2d(1.91, 2.29), Rotation2d.fromDegrees(35.04));
+  private static final byte[] STRUCT_BUFFER =
+      new byte[] {
+        -113, -62, -11, 40, 92, -113, -2, 63, 82, -72, 30, -123, -21, 81, 2, 64, -41, -77, 36, 88,
+        -19, -111, -29, 63
+      };
```

Comment:
Is there a way for the connection between these bytes and the expected double values to be better explained? An idea would be using a static initializer to pack values into a ByteBuffer that is in some way associated with the array. If that can't be done, at the very least using hexidecimal literals for doubles would be better since we don't care about the numeric values of the individual bytes.